### DISCUSSION TITLE
Support tags attributes within translations.

### DISCUSSION THREAD

**Discussion by wscourge** [amannn/next-intl#615] [Q&A]
### Is your feature request related to a problem? Please describe.

I'd like a default translation value for the `<abbr />` element, that requires the `title` attribute:

```html
<abbr title="HyperText Markup Language">HTML</abbr>
```

Other, similar use-case: tooltips and anchor titles.

### Describe the solution you'd like

```
{
  "message": "Read more about: <abbr title='HyperText Markup Language'>HTML</abbr>."
}
```

### Describe alternatives you've considered

This [Docs / Usage / Messages / Rich Text Q&A: How can I pass attributes to tags](https://next-intl-docs.vercel.app/docs/usage/messages#rich-text). I however do not see it solving the "default" problem.

**Comment by amannn** [comment]
I understand the request, but the ICU parser used by `next-intl` currently doesn't support attributes. See "How can I pass attributes to tags?" in [the rich text guide](https://next-intl-docs.vercel.app/docs/usage/messages#rich-text).

I'll move this to a discussion since it seems to be a usage question.

**Reply by wscourge** [reply]
I'm aware of the guide, yet I don't know how to solve the "default" problem using it - can I?

If not, let me know:

> The ICU parser used by `next-intl`

which dependency is it and I'll request this feature upstream.

**Reply by alkorlos** [reply]
@wscourge How did you resolve this issue?
Did you create an issue in the parser library? Or did you use a different library for internationalization?

**Reply by wscourge** [reply]
I didn't resolve it. If you open issue in the parser lib, please let me know, I'll happily upvote it

**Reply by alkorlos** [reply]
Issue has already been made, it is closed, and there will be no implementation
https://github.com/formatjs/formatjs/issues/4366
